src/main/java/org/squiddev/cobalt/debug/DebugState.java

```diff
@@ -129,7 +133,8 @@ public DebugFrame pushInfo() throws LuaError {
 		DebugFrame[] frames = stack;
 		int length = frames.length;
 		if (top >= length) {
-			if (top >= MAX_SIZE) throw new LuaError("stack overflow");
+			if (top >= MAX_SIZE)
```

Comment:
\>_>

Reply:
Haha, conditional breakpoints are really expensive, so I split this into two lines when I needed to debug a stack overflow. I'll revert the change if you want to.

Reply:
Yes please. Sorry - multi-line ifs without braces are evil!